datumaro/plugins/open_images_format.py

```diff
@@ -175,7 +175,10 @@ def __init__(self, path, image_meta=None):
 
         self._dataset_dir = path
 
-        self._annotation_files = os.listdir(osp.join(path, OpenImagesPath.ANNOTATIONS_DIR))
+        self._annotation_dir = osp.join(path, OpenImagesPath.ANNOTATIONS_DIR)
```

Comment:
Please decide if you want to support different layouts and document, if so.

Reply:
[Decision already applied](https://github.com/openvinotoolkit/datumaro/blob/7676f2e42ce2e65351c7a2b15eecc0ec0aeecb90/datumaro/plugins/open_images_format.py#L178-L180): if `annotations` directory exists then search annotation file in it, else search annotation file in root of input path.